examples/nextjs-example/pages/index.tsx

```diff
@@ -49,86 +51,98 @@ export default function Home() {
         defaultFeedOptions={{ tenant }}
       >
         <NotificationFeedContainer>
-          <Box maxW="520px" mx="auto" py={12}>
-            <Flex mb={8}>
-              <Box>
-                <Heading size="md" mb={2}>
+          <Box style={{ maxWidth: "500px" }} mx="auto" py="12">
```

Comment:
Nit: Ideally we use telegraph constants wherever possible. In the case of a demo app like this, I think just setting a reasonable pixel value is totally okay but wanted to still call it out since having to use `style` with telegraph is often a smell.

Reply:
I wanted to match the current example as best as possible and telegraph didn't over a suitable `maxWidth` to use here. I tried!